src/ssl.c

```diff
@@ -5415,6 +5415,12 @@ int wolfSSL_Init(void)
 
     WOLFSSL_ENTER("wolfSSL_Init");
 
+#ifdef WOLFSSL_CHECK_MEM_ZERO
+    /* Initialize the mutex for access to the list of memory locations that
+     * must be freed. */
+    wc_MemZero_Init();
```

Comment:
Would this be better in `wolfCrypt_Init()`?

Reply:
Yep! Fixed.